CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 [comment]: <> "NOTE! Ensure all images are added via the \[label\]\(link\) syntax!"
 
+## 2025-04-24 New Operation Insights Errors page
+| What's new | |
+| :--------- | :-: |
+| We've introduced a brand new Errors Insights page for enhanced error telemetry including categorization by service and error code. See our [documentation](https://www.apollographql.com/docs/graphos/platform/insights/errors) for more detailed information and instructions on how to enable an even richer experience with [extended error reporting](https://www.apollographql.com/docs/graphos/platform/insights/errors#extended-error-reporting).| ![image](https://github.com/user-attachments/assets/70d018ea-bdbc-479d-9e1a-c07aab246536)
```

Comment:
Stylistic suggestions to:
- Make it more concise
- Make it clearer what's changed

```suggestion
| The Error > Insights page now lets your group and filter errors by underlying service and error code. See the [documentation](https://www.apollographql.com/docs/graphos/platform/insights/errors) for more information and instructions on how to enable [extended error reporting](https://www.apollographql.com/docs/graphos/platform/insights/errors#extended-error-reporting).| ![image](https://github.com/user-attachments/assets/70d018ea-bdbc-479d-9e1a-c07aab246536)
```